### ISSUE TITLE
ios Crash

### ISSUE THREAD

**Issue by qq326646683** [Sh1d0w/multi_image_picker#256]
**Bug Report**

`multi_image_picker` version: 4.5.3

Is it happening on Android: No

Is it happening on iOS: Yes

```
[✓] Flutter (Channel stable, v1.7.8+hotfix.4, on Mac OS X 10.14.3 18D42, locale
    zh-Hans-CN)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 10.2.1)
[✓] iOS tools - develop for iOS devices
[✓] Android Studio (version 3.4)
[✓] IntelliJ IDEA Ultimate Edition (version 2019.1.3)
[✓] Connected device (1 available)

• No issues found!
```

**Describe the bug**

<img width="1102" alt="crash1" src="https://user-images.githubusercontent.com/15717688/64915371-a03e9400-d797-11e9-8e20-27b3361cf5d4.png">
<img width="1051" alt="crash2" src="https://user-images.githubusercontent.com/15717688/64915372-a3d21b00-d797-11e9-8271-777ca4ba5aee.png">

**Comment by Sh1d0w** [comment]
Please provide more info, what is the exact error message, when it happens, how to reproduce etc.

From those screenshots nothing from that becomes clear.

**Comment by qq326646683** [comment]
hi, this bug still exist, after select,click done, app crash
![ios_crash_multi_photo](https://user-images.githubusercontent.com/15717688/65616054-f8be2e80-dfec-11e9-9f34-0cc4ffb7c098.png)

**Comment by Sh1d0w** [comment]
I will check this, thanks for the screenshot.

**Comment by Sh1d0w** [comment]
Hey @qq326646683 I think it should be fixed in version `4.5.6` . Can you try that out?

**Comment by qq326646683** [comment]
@Sh1d0w thank you, it has fixed.
